packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
 $uikit-colour-Primary: #880e48 !global;
 $uikit-colour-Background: mix(darken(#f0f3f5, 40), #f0f3f5, 50);
-$uikit-colour-Border: mix(black, #bebebe, 50) !global;
+$uikit-colour-Border: #bbbbbb !global;
```

Comment:
This was the colour used in uikit 1.x and makes our borders the correct colour.

In making this edit we also fix body’s `hr` (which uses this colour via the var).